net-im/swiften/swiften-4.0.ebuild

```diff
@@ -0,0 +1,120 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit scons-utils toolchain-funcs
+
+DESCRIPTION="A C++ library for implementing XMPP applications"
+HOMEPAGE="https://www.swift.im/"
+SRC_URI="https://swift.im/downloads/releases/swift-${PV}/swift-${PV}.tar.gz"
+
+LICENSE="BSD BSD-1 CC-BY-3.0 GPL-3 OFL-1.1"
+SLOT="0"
```

Comment:
doesn't that need a subslot?

Reply:
That's a good question. What about /usr/bin/swiften-config? That would collide in a multi slot?

Reply:
no normal slot, just a subslot so portage will rebuild it when SONAME changes (e.g on a swiften 5 bump)
the packages depending on it will need a subslot dep as well.

Reply:
Thanks, I didn't have thought on the subslot. Added a one now.